app/scripts/first-time-state.js

```diff
@@ -10,6 +10,17 @@
  */
 const initialState = {
   config: {},
+  PreferencesController: {
```

Comment:
Ni: It would seem safer to add this in the migration, rather than here. Anything added here will be subject to all of the migrations for new installations.

If we've tested this specific addition and it's OK, then maybe this is non-blocking, but this isn't a pattern we should follow often I don't think.